packages/sdk/src/__tests__/judge-e2e.test.ts

```
@@ -60,4 +60,63 @@ describe("judge mode e2e", () => {
       process.chdir(prev);
     }
   });
+
+  it("normalizes fenced JSON judge output before writing artifacts", async () => {
+    const root = await mkdtemp(join(tmpdir(), "obora-judge-fenced-"));
+    await mkdir(join(root, "artifacts"), { recursive: true });
+    await writeFile(
+      join(root, "artifacts", "submission.json"),
+      JSON.stringify({ title: "Sample", body: "Clear answer", rubric: { clarity: 1 } }, null, 2),
+      "utf8",
```

ESLint

[error] Delete `,`

(prettier/prettier)